codex-rs/cli/src/debug_sandbox/pid_tracker.rs

```diff
@@ -288,8 +288,8 @@ mod tests {
     }
 
     #[cfg(target_os = "macos")]
-    #[test]
-    fn list_child_pids_includes_spawned_child() {
+    #[tokio::test]
+    async fn list_child_pids_includes_spawned_child() {
         let mut child = Command::new("/bin/sleep")
```

Comment:
Should we use `tokio::process::Command`, as well if we're going to make the test async?